docs/features/traffic-management/istio.md

```diff
@@ -84,6 +85,7 @@ spec:
 
 Finally, a canary and stable Service should be deployed. The selector of these Services will be
 modified by the Rollout during an update to target the canary and stable ReplicaSet pods.
+Note that if virtualservice and destionation host reside in different namespaces (e.g., virtualservice and rollout are not in the same namespace), we should use FQDN as the destination host like `stable-svc.<namespace>`.
```

Comment:
1. Nit: Typo in destination
2. Is it always true that if that virtualservice and rollout are not in the same namespace, the virtualservice and destination host reside in different namespaces? I think it is more likely that the virtualservice and service are in the same namespace